apps/settings/test/unit/panels/root/stk_item_test.js

```diff
@@ -86,10 +87,6 @@ suite('STK (Main menu) >', function() {
           }})});
     });
 
-    teardown(function() {
-      mocksHelper.teardown();
-    });
-
     test('One entry into the STK applications list', function() {
       assert.equal(document.getElementById('icc-entries').childElementCount, 1);
     });
```

Comment:
Can you add some test suites to make sure we did render icon correctly and can be access from the DOM here ? (Including the failing case)